app/decorators/controllers/solidus_bolt/spree_checkout_controller/add_addresses_to_bolt.rb

```diff
@@ -18,6 +21,13 @@ def finalize_order
       end
 
       Spree::CheckoutController.prepend self
+
+      private
+
+      def session_refreshed_or_expired?
```

Comment:
Considering we are not refreshing the access_token anymore, maybe it will be better to rename it to `write_access_token_expired?` or something like this.

Reply:
fixed in https://github.com/solidusio-contrib/solidus_bolt/pull/112/commits/57a97b70284a7c34e97420d65661158ac925c43f